packages/app-react/@types/web3-provider.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@walletconnect/web3-provider';
```

Comment:
This is strange, as the package repository seems to have the file but not the npm package :/

Reply:
Yea they don’t have proper types in there either, so it’s not really typesafe either way, this is just to silence the warning in strict mode